reconcile/test/utils/dynatrace/conftest.py

```diff
@@ -20,7 +20,21 @@ def builder(data: Mapping[str, Any]) -> Dynatrace:
             token.token = token_created_result
             token_service.create.return_value = token
         else:
+            # For raising exceptions
             token_service.create.side_effect = token_created_result
+
+        list_token_result = data.get("LIST_TOKEN_RESULT", [])
+        if isinstance(list_token_result, list):
+            listed_tokens = []
+            for token_tup in list_token_result:
+                token = create_autospec(spec=ApiTokenCreated)
+                token.id = token_tup[1]
+                token.name = token_tup[0]
```

Comment:
is there a field called `name` in `ApiTokenCreated`? `list` should return `ApiToken`

Reply:
Oh copy pasta. Fixed now. Nice catch! :+1: